app/src/main/java/org/mozilla/tv/firefox/pocket/PocketViewModel.kt

```diff
@@ -23,11 +27,15 @@ const val POCKET_VIDEO_COUNT = 20
  * view should not have to perform any transformations on this data).
  */
 class PocketViewModel(
+    resources: Resources,
```

Comment:
Alternatively, we could extend `AndroidViewModel()`

Reply:
Can `AndroidViewModel` classes be tested without Robolectric?  I haven't checked.  Cause I do like that `Resources` can be easily mocked without using Robo, since it has such a dramatic impact on test start times.